sim/src/Manager.h

```diff
@@ -0,0 +1,46 @@
+#ifndef BENTOBOX_MANAGER_H
```

Comment:
`Manager.h` seems  part of ECS, should it be under `ics/`?

Reply:
It is part of ICS, but I can't put it in `ics/` because it uses `typeMap.h`. So, to show the dependencies clearly and because of CMake, I have put it outside.

I can rename it to `ICSManager`, though I am planning to create another namespace for the ICS implementations

Reply:
This seems be a symptom of our build system being misconfigured. Lets put this off for now.

> I can rename it to ICSManager, though I am planning to create another namespace for the ICS implementations

`ICSManager` is much clearer.

Reply:
Fixed in c44326c and 3ef67ff